_rules/SC4-1-2-iframe-unique-name.md

```diff
@@ -0,0 +1,294 @@
+---
+name: Iframes with identical accessible names serve equivalent purpose
+
+description: |
+ This rule checks that iframes with identical accessible names serve an equivalent purpose
+
+success_criterion:
+- 4.1.2
+
+test_aspects:
+- DOM Tree
+- CSS Styling
+
+authors:
+- Jey Nandakumar
+- Audrey Maniez
+---
+
+## Test procedure
+
+### Applicability
+
+This rule applies to `iframe` elements that are [included in the accessibility tree](#included-in-the-accessibility-tree) and have [matching](matching-characters) [accessible name](#accessible-name), that do not only include whitespace.
```

Comment:
I still suggest aligning with https://github.com/auto-wcag/auto-wcag/pull/220, which would mean this applicability:

> This rule applies to any set of any two or more `iframe` elements that are [included in the accessibility tree](#included-in-the-accessibility-tree), and that have [matching](#matching-characters) [accessible names](#accessible-name) that does not only consist of [whitespace](#whitespace).

"whitespace" needs to link to the definition anyway.